psql_authnz/psql_authnz.py

```diff
@@ -30,7 +31,7 @@ def main():
 
     # Setup logging
     LOG_FORMAT = "%(asctime)-15s PSQL-AUTHNZ [%(levelname)-5s]: %(filename)-15s:%(lineno)-3s - %(message)s"
-    logging.basicConfig(format=LOG_FORMAT,level=getattr(logging, log_level.upper()))
+    logging.basicConfig(filename=log_file,format=LOG_FORMAT,level=getattr(logging, log_level.upper()))
```

Comment:
Thanks!  In Python, passing "None" to a keyword argument is effectively the same as not supplying one at all.